presto-main/src/main/java/com/facebook/presto/sql/planner/plan/ExchangeNode.java

```diff
@@ -120,13 +120,29 @@ public static ExchangeNode partitionedExchange(PlanNodeId id, Scope scope, PlanN
     }
 
     public static ExchangeNode partitionedExchange(PlanNodeId id, Scope scope, PlanNode child, List<Symbol> partitioningColumns, Optional<Symbol> hashColumn, boolean replicateNullsAndAny)
+    {
```

Comment:
We have multiple `partitionedExchange` methods with complex list of arguments, would be nice to unify some of them to simplify the API surface here.

Reply:
@nezihyigitbasi : Does the next commit (rename some of them specifically to `systemPartitiontedExchange` ) help to clarify the API ?